pkg/grep.go

```diff
@@ -28,6 +31,17 @@ func NewGrep(expr string) (*Grep, error) {
 				equal = false
 			}
 
+			wildcardPrefix := false
```

Comment:
Instead of specific `wildcardPrefix` & `wildcardSuffix`, would it make sense / work / be more flixble if this implementation would be made such that simple wildcard `*` would be replaced to regexp match all `.*` and then matched with regexp?

https://play.golang.org/p/2z8ceQ1AVxV

I believe this would be simpler and a bit more flexible (though, the whole grep value as such could be plain regexp as well).

Reply:
I thought performance reasons, but I agree this is BS. I'm using this branch for a while now and I'm happy with using wildcards. Therefore I don't want spend too much time on it. Is it good enough?

PS. Feel free to contribute and introduce regexp or even hijack this PR.

Reply:
I'll provide new PR when I need further functionality. I'm good with this.